crates/evm-ee/src/types/block.rs

```diff
@@ -42,3 +47,70 @@ impl ExecBlock for EvmBlock {
         &self.body
     }
 }
+
+impl Encode for EvmBlock {
+    fn is_ssz_fixed_len() -> bool {
+        false
+    }
+
+    fn ssz_bytes_len(&self) -> usize {
+        // SSZ container with 2 variable-length fields:
+        // - Fixed part: 2 offsets (4 bytes each) = 8 bytes
+        // - Variable part: header bytes + body bytes
+        let header_len = self.header.ssz_bytes_len();
+        let body_len = self.body.ssz_bytes_len();
+        8 + header_len + body_len
+    }
+
+    fn ssz_append(&self, buf: &mut Vec<u8>) {
+        // SSZ container encoding: write offsets first, then variable data
+        let header_len = self.header.ssz_bytes_len();
+
+        // Fixed part: offsets (4 bytes each, little-endian)
+        let header_offset = 8u32; // After the 2 offsets
+        let body_offset = (8 + header_len) as u32;
+        buf.extend_from_slice(&header_offset.to_le_bytes());
+        buf.extend_from_slice(&body_offset.to_le_bytes());
+
+        // Variable part: header and body data
+        self.header.ssz_append(buf);
+        self.body.ssz_append(buf);
+    }
+}
+
+impl Decode for EvmBlock {
```

Comment:
Don't really like this ad-hoc manual encode/decode.

SSZ isn't providing us much value here since the things we'd want to make SSZ proofs about (txs, receipts, etc.) are contained in an RLP structure, so we couldn't do that anyways.  I think this was fine being `Codec` as-is.